frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx

```diff
@@ -284,7 +288,8 @@ export const BuildingReviewPage: React.FC<any> = (props: IReviewProps) => {
                   disabled={editInfo.valuation}
                 />
                 <Input
-                  field="data.financials.0.netbook.fiscalYear"
+                  field="netbookYearDisplay"
```

Comment:
Is this ok to have a dummy field for display only? Or should I make a separate component that does not require field etc